libtsuba/src/RDGManifest.cpp

```diff
@@ -286,7 +287,7 @@ katana::RDGManifest::FileNames() {
       }
 
       for (auto it : header.optional_datastructure_manifests()) {
-        fnames.emplace(it.first);
+        fnames.emplace(it.second);
```

Comment:
These kinds of bugs are harder to make if we have wrappers around the iterator type which return the desired value. e.g: FileNameFrom...

Reply:
Dang